addons/binding/org.openhab.binding.km200/src/main/java/org/openhab/binding/km200/KM200ServiceType.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.km200;
+
+/**
+ * The KM200ServiceTypes enum is representing the known main service paths
+ *
+ * @author Markus Eckhardt
+ *
+ */
+public enum KM200ServiceType {
+    DHWCIRCUITS {
+        @Override
+        public String getRootPath() {
```

Comment:
There is imho a better way to do this, by creating a constructor for this enum like

```java
DHWCIRCUITS("/dhwCircuits")...;

private final String rootPath;

KM200ServiceType(String rootPath) {
    this.rootPath = rootPath;
}

public String getRootPath() {
    return rootPath;
}
```

Reply:
I changed the enums and the handling. All data are now in one enum and it is extended with the contructor and default data handling. Please take a look....